elisp/intero.el

```diff
@@ -263,6 +263,27 @@ This is slower, but will build required dependencies.")
 (defvar-local intero-buffer-host nil
   "The hostname of the box hosting the intero process for the current buffer.")
 
+(defmacro intero-inherit-local-variables (buffer)
+  "Make the current buffer inherit values of certain local variables from BUFFER."
+  (let ((varlist '(intero-stack-executable
+                   intero-repl-no-build
+                   intero-repl-no-load
+                   ;; TODO: shouldn’t more of the above be here?
+                   )))
+    `(progn
+       ,@(cl-mapcar
+          (lambda (symb)
+            `(setq-local ,symb (buffer-local-value (quote ,symb) ,buffer)))
+          varlist))))
```

Comment:
I feel like this doesn't need to be a macro. If you thought it necessary in order to appease `setq-local`, then `(set (make-local-variable 'the-sym) value)` would have the same effect.